package.json

```diff
@@ -78,7 +79,7 @@
     "chalk": "^1.1.3",
     "clone": "^2.0.0",
     "cssbeautify": "^0.3.1",
-    "doctrine": "^0.7.0",
+    "doctrine": "^2.0.0",
```

Comment:
I'm surprised this didn't cause any issues, as there've been a bunch of changes. Over a year of active development. Good to be up to date!